planetarium/NineChronicles
# Improve `ConditionalCostButton` to handle various currencies.

### Description

SSIA

### How to test

1. Check if `ConditionalCostButton`s are still working properly.
2. Check if `OnClickSubject` is called on `Conditional` state in `ConditionalCostButton`

### Required Reviewers
@sonohoshi
